contracts/vaults/Pool.sol

```diff
@@ -108,36 +96,19 @@ contract Pool is Base {
     /**
     * @dev setPool
     * @param _pid the pool id
-    * @param _allocPoint the pool allocation point
     * @param _visible is this pool visible in the UI
     * @param _depositPause pause pool deposit (default false).
     * This parameter can be used by the UI to include or exclude the pool
     * @param _descriptionHash the hash of the pool description.
     */
     function setPool(uint256 _pid,
-                    uint256 _allocPoint,
                     bool _visible,
                     bool _depositPause,
                     string memory _descriptionHash)
     external onlyOwner {
         require(poolInfos.length > _pid, "HVE23");
-        updatePool(_pid);
```

Comment:
so this is only used now to set _depositPause? then we should rename

Reply:
so this is a bit of a confusing pattern but there is reason to it. So as you may have noticed, "_visible" and "_descriptionHash" are never ever SET anywhere. The only thing that happens is that we emit an event with this info. This event can then be picked up by the UI, which can show it. 

This save some gas, while still keepig a trace on-chain, and using the on-chain security. So setPool can stay